src/Application/Application.csproj

```diff
@@ -9,11 +9,16 @@
     <PackageReference Include="Ardalis.GuardClauses"/>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection"/>
     <PackageReference Include="FluentValidation.DependencyInjectionExtensions"/>
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" />
```

Comment:
We can use SignalR here, instead we can create interface and implement this interface in Infrastructure layer